src/voucherify/voucherify-connector.service.ts

```diff
@@ -12,6 +13,8 @@ import {
   REQUEST_JSON_LOGGER,
 } from '../misc/request-json-logger';
 import { Coupon } from 'src/api-extension/coupon';
+import { SimpleCustomer } from '@voucherify/sdk/dist/types/Customers';
```

Comment:
Consider check imports usages. I suppose that some of them are not used in any case.